src/main/java/transportservice/ExtensionSettings.java

```diff
@@ -0,0 +1,39 @@
+package transportservice;
+
+public class ExtensionSettings {
+
+    private String extensionname;
+    private String hostaddress;
+    private String hostport;
+    static final String EXTENSION_DESCRIPTOR = "src/main/java/transportservice/extension.yml";
```

Comment:
Do we need the full path ?
Thinking out loud, this should something configurable by the extension when they integrate with the SDK. 

And move the `extension.yml` to test resources, because its only needed for us to test. But when the extension is integrated, this path is not valid any more.
What do you think ?

Also @joshpalis as our first customer, how does this look?

Reply:
That's a great point. Should I keep `EXTENSION_DESCRIPTOR` empty for now and move extension.yml to test/resources? And use the yml file only for testing SDK locally

Reply:
Yeah that would be perfect.

Reply:
Pushed the change. Thanks for the review